lmnet/lmnet/datasets/bdd100k.py

```diff
@@ -1,16 +1,15 @@
 # -*- coding: utf-8 -*-
-import functools
```

Comment:
Could you apply `isort` to sort import orders?

Reply:
Done